tests/unit/h/schemas/api/group_test.py

```diff
@@ -13,167 +13,168 @@
     UpdateGroupAPISchema,
 )
 
+DEFAULT_AUTHORITY = "hypothes.is"
 
-class TestGroupAPISchema:
-    def test_it_sets_authority_properties(self, third_party_schema):
-        assert third_party_schema.group_authority == "thirdparty.com"
-        assert third_party_schema.default_authority == "hypothes.is"
```

Comment:
I'm glad to see this `third_party_schema` fixture gone: it's not obvious from the fixture name what it actually is (an instance of `GroupAPISchema` where `GroupAPISchema.group_authority` is `"thirdparty.com"` but `default_authority` is still `"hypothes.is"` the same as the `schema` fixture). It's clearer for tests just to create `GroupAPISchema` instances with different properties directly.